ibmcloud/IMPORT_PODVM_TO_VPC.md

```diff
@@ -0,0 +1,38 @@
+# Importing Public PODVM images into IBM Cloud VPC
+
+As part of the release process pre-built images are published as container images to the confidential-containers quay repository. e.g. `quay.io/confidential-containers/podvm-ibmcloud-ubuntu-s390x`. These contain a single qcow2 file that needs to be extracted and uploaded to ibmcloud to use as a vpc image.
+
+To simpify this process a script has been created to aid this. `ibmcloud/image/import.sh`.
+
+## Prerequisites
+
+### Tools
+
+- jq `apt install jq`
+- ibmcloud `curl -fsSL https://clis.cloud.ibm.com/install/linux | sh` (https://cloud.ibm.com/docs/cli?topic=cli-getting-started)
+- docker/podman `apt install docker.io`
+
+### Cloud-Object-Storage
+
+To create the VPC image you need to first import the file to ibmcloud COS. The script will do this step but a bucket must already be available.
+
+You may follow the [offical documentation](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-getting-started-cloud-object-storage) to create a bucket, a free tier is sufficient.
+
+## Running
+
+### Example
+
+`./import.sh quay.io/confidential-containers/podvm-ibmcloud-ubuntu-s390x ca-tor --instance jt-cos-instance --bucket podvm-image-cos-bucket-jt --region jp-tok`
```

Comment:
As mentioned, I did not have access to this specific image. Although this is just an example, should it instead be pointing at: `quay.io/confidential-containers/podvm-ibmcloud-ubuntu`

Reply:
The arch suffix is part of the plan for later builds